cloud-init.yaml

```diff
@@ -0,0 +1,37 @@
+#cloud-config
+
+packages:
+  - podman
+  - podman-compose
+
+snap:
+  commands:
+    0: [install, core]
+    1: [refresh, core]
+    2: [install, --classic, certbot]
```

Comment:
Note the user of certbot, that means we need to do https://github.com/holochain/kitsune2/issues/136 in the next month or two

Reply:
Ideally sooner, so we don't forget :)